services/indexes/avm/writer.go

```diff
@@ -89,29 +93,29 @@ func NewWriter(networkID uint32, chainID string) (*Writer, error) {
 
 func (*Writer) Name() string { return "avm-index" }
 
-func (w *Writer) initCtx(tx *avm.Tx) {
-	switch castTx := tx.UnsignedTx.(type) {
-	case *avm.CreateAssetTx:
-		for _, utxo := range castTx.UTXOs() {
+func (w *Writer) initCtx(tx *txs.Tx) {
```

Comment:
This function shouldn't exist anymore

Reply:
I'm gathering from the context that all of these txs are pre-initialized.  But can you please confirm